src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -366,155 +370,237 @@ A working alternative would be using an icon together with an `aria-label`:
 	}
 	</script>
 ```
-</docs>
 
+### Conditionally show the sidebar
+
+If the sidebar should be shown conditionally (e.g. using a button)
+and the users are expected to open and close the sidebar multiple times,
+then using `v-if` might result in bad performance.
+So instead use the `open` property:
+
```

Comment:
Can you also mention that when `open` prop is used, there is a built-in toggle button, you users don't have to implement it manually?

Reply:
Done :white_check_mark: